todo-frontend-angular/src/app/todolist-selector/todolist/sort.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Todo } from 'src/app/models/todo.model';
+
+@Pipe({
+  name: 'sort'
+})
+
+export class SortPipe implements PipeTransform {
+  transform(todos: any[] | null, field: string): any[] {
```

Comment:
Uhhh not the any :O use unknown instead or maybe you should how to make a Generic pipe :D so it would be `transform(todos: T[] | null, field: keyof T)`

Reply:
fixed